jenkins/attach_agent.py

```diff
@@ -14,10 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Attach Jenkins agents."""
 
 import ConfigParser
 import sys
 
+# pylint: disable=import-error
```

Comment:
Hmm... Can you put a TODO to fix this? We sould be able to create a py_binary bazel target that has the jenkinsapi dep. Alternatively should we add this to the image we use for running the verify check (like we do requests)?